01-atoms/buttons/buttons.stories.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import Button from './Button.component';
+
+/**
+ * Storybook Definition.
+ */
+export default { title: 'Atoms/Button' };
+
+export const All = () => (
+  <>
+    <Button>Button Default</Button>
+    <br />
+    <Button modifiers={['alt']}>Button Alt</Button>
+    <br />
+    <Button modifiers={['alt-2']}>Button Alt 2</Button>
```

Comment:
No need to redeclare these. They are exported below. We can avoid redundant code.
```
    <Default />
    <br />
    <Alt />
    <br />
    <Alt2 />
```